README.md

```diff
@@ -2,4 +2,76 @@
 
 # thinc-apple-ops
 
-Make [thinc](https://thinc.ai) faster on MacOS by calling into Apple's native libraries.
+Make [thinc](https://thinc.ai) faster on macOS by calling into Apple's native
+libraries.
+
```

Comment:
We should probably have a quick summary here or repeat the benchmarks (or a smaller version of them) here so people don't have to scroll to get the juicy numbers?